### PR TITLE
Start using visualization modal URLs

### PR DESCRIPTION
**Pull request by xavijam**
After trying to fix https://github.com/CartoDB/cartodb/issues/8116, I've realized that shared datasets (in an organization) can't be used in the builder, we should take a look after this week. This could be a workaround for now.

cc @CartoDB/frontend.

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/data/vis-definition-model.js [hunk 1]
```diff
@@ -34,6 +34,11 @@ module.exports = Backbone.Model.extend({
     return baseUrl + '/builder/' + this.id + '/embed';
   },
 
+  builderURL: function () {
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/data/vis-definition-model.js]: 👍  we must use this in the share modal.

**Comment by xavijam** [comment]
@jvillarf on-premises tests failing, should we be worried?

**Comment by jvillarf** [comment]
Will take a look :)

**Comment by jvillarf** [comment]
@xavijam again, there was a connectivity issue with registry.npmjs.org, the IP changed... :(

I've added the new IP to the firewall, so you might be able to re-test it.

Javi.

**Comment by xavijam** [comment]
Merging until we start using better URL management.

**Pull request merged**